### ISSUE TITLE
The Fingerprint HAL is not in the device tree...

### ISSUE THREAD

**Issue by penn5** [racer123/android_device_nokia_PLE#7]
There is simply no reference to it. No biometrics folder, no fingerprint folder. I don't see how fingerprint could work like this. This might be useful: https://github.com/HridayHS/android_device_xiaomi_land/commit/6e7d2449713d65ad9c3614c54d38f8dca64c501f

**Comment by gssdromen** [comment]
@penn5 is this device tree complete? I built lineageOS 15.1 with this device tree, a lot of things does not work....

**Comment by tboHammu** [comment]
@gssdromen it's work in progress, most things work but some do not.

**Comment by gssdromen** [comment]
@racer123 Do you have a flashable zip? Wifi and Navigate Buttons do not work in my build. Do I need to flash a specific version with OST LA first?

**Comment by tboHammu** [comment]
@gssdromen No, I don't at the moment. And I don't think it matters which version you are on before flashing. I'll probably do an alpha release once I get the speakerphone working.

**Comment by gssdromen** [comment]
@racer123 thanks for reply, looking forward to your release version
